### PR TITLE
Updated Save As to save the Script window contents, also made the File>Save Click functional

### PR DESCRIPTION
**Pull request by deaspo**
Updated Save As to save the Script window contents, also made the File>Save Click functional

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -29,6 +29,7 @@ Public Class frmMain
     Public clsInstatOptions As InstatOptions
     Public strCurrentDataFrame As String
     Public dlgLastDialog As Form
+    Public iSaveCount As Integer = 0
```
Comment (dannyparsons) [inline instat/frmMain.vb]: What is the purpose of the Save count?

Comment (deaspo) [inline instat/frmMain.vb]: I have made some updates, removed the iSaveCount.

File: instat/dlgSaveAs.vb [hunk 2]
```diff
@@ -1,7 +1,52 @@
...
+    Public Function SaveDialog(strFilter As String) As KeyValuePair(Of String, String)
+        Dim dlgOpen As New SaveFileDialog
+        Dim strFilePath, strFileName As String
+        Select Case strFilter
+            Case ".txt"
+                dlgOpen.Filter = "Text file (*.txt)|*.txt"
+            Case ".rtf"
+                dlgOpen.Filter = "Rich Text file (*.rtf)|*.rtf"
+            Case ".dataFile"
+                dlgOpen.Filter = "Comma separated file (*.csv)|*.csv|RDS (*.RDS)|*.RDS"
```
Comment (dannyparsons) [inline instat/dlgSaveAs.vb]: Is there a reason to put these two cases together? I feel they should be separate since they save different things.

**Comment by deaspo** [comment]
Hi @dannyparsons , sorry for the initial pull request. I was using an old version. Have updated the pull request

**Pull request merged**